view/lang/pl/messages.po

```diff
@@ -7436,7 +7436,7 @@ msgstr "Jeśli chcesz %s odpowiedzieć, sprawdź, czy ustawienia prywatności w
 msgid ""
 "This is Friendica, version %s that is running at the web location %s. The "
 "database version is %s, the post update version is %s."
-msgstr ""
+msgstr "To jest wersja Friendica, %s która działa w lokalizacji internetowej %s. Wersja bazy danych to %s wersja po aktualizacji%s."
```

Comment:
I guess there is a space missing in front of the last %s - for me it looks like some small translation problem.

Reply:
Just change it at Transifex ;-)